test/python/circuit/test_controlled_gate.py

```diff
@@ -886,6 +886,40 @@ def test_open_controlled_gate_raises(self):
         with self.assertRaises(CircuitError):
             base_gate.control(num_ctrl_qubits, ctrl_state='201')
 
+    def test_base_gate_params_reference(self):
+        """
+        Test all gates in standard extensions which are of type ControlledGate and have a base gate
+        setting have params which reference the one in their base gate.
+        """
+        num_ctrl_qubits = 1
+        for gate_class in ControlledGate.__subclasses__():
+            with self.subTest(i=repr(gate_class)):
+                num_free_params = len(_get_free_params(gate_class.__init__, ignore=['self']))
+                free_params = [0.1 * i for i in range(num_free_params)]
```

Comment:
It's probably better to not have the first parameter be 0, this might be a special case and not representative (e.g. before, some tests failed because the global phase was 0 by accident, as the rotation angle in the tests was 0).
```suggestion
                free_params = [0.1 * (i + 1) for i in range(num_free_params)]
```